SHTSensor.h

```diff
@@ -32,6 +32,18 @@
 #include <inttypes.h>
 #include <Wire.h>
 
+//#define DEBUG_SHT_SENSOR
+
+#ifdef DEBUG_SHT_SENSOR
+#ifdef DEBUG_ESP_PORT
+#define DEBUG_SHT(f,...) do { DEBUG_ESP_PORT.printf(PSTR(f), ##__VA_ARGS__); } while (0)
+#else
+#define DEBUG_SHT(f,...) do { Serial.printf(PSTR(f), ##__VA_ARGS__); } while (0)
```

Comment:
Serial only has print() and println(), no printf: https://www.arduino.cc/reference/en/language/functions/communication/serial/

Reply:
ESP8266 Arduino supports it ...
https://github.com/esp8266/Arduino/blob/5e0ed3a38bbe9a1e203e39eb4734dc236c41e57e/cores/esp8266/Print.h#L82C17-L82C17
But makes no sense using it, if other variants do not. Wasn't aware of that. At the end of the day I do not use parameter at the moment, so just using print is fine. Will fix this.

Reply:
Ok, done with [fbadd08](https://github.com/Sensirion/arduino-sht/pull/29/commits/fbadd08e6c561f99e26c0298d77e253c6613530a).